src/persist/src/indexed/runtime.rs

```diff
@@ -619,40 +664,40 @@ mod tests {
     }
 
     #[test]
-    fn atomic() -> Result<(), Error> {
+    fn multi() -> Result<(), Error> {
         let data = vec![
             (("key1".to_string(), ()), 1, 1),
             (("key2".to_string(), ()), 1, 1),
         ];
 
         let mut registry = MemRegistry::new();
-        let client1 = registry.open::<String, ()>("1", "atomic")?;
-        let client2 = registry.open::<String, ()>("2", "atomic")?;
+        let client1 = registry.open::<String, ()>("1", "multi")?;
+        let client2 = registry.open::<String, ()>("2", "multi")?;
 
         let (c1s1, c1s1_read) = client1.create_or_load("1")?;
         let (c1s2, c1s2_read) = client1.create_or_load("2")?;
         let (c2s1, _) = client2.create_or_load("1")?;
 
         // Cannot construct with no streams.
-        assert!(AtomicWriteHandle::<(), ()>::new(&[]).is_err());
+        assert!(MultiWriteHandle::<(), ()>::new(&[]).is_err());
 
         // Cannot construct with streams from different runtimes.
-        assert!(AtomicWriteHandle::new(&[&c1s2, &c2s1]).is_err());
+        assert!(MultiWriteHandle::new(&[&c1s2, &c2s1]).is_err());
 
         // Normal case
-        let atomic = AtomicWriteHandle::new(&[&c1s1, &c1s2])?;
+        let multi = MultiWriteHandle::new(&[&c1s1, &c1s2])?;
         let updates = vec![
             (c1s1.stream_id(), data[..1].to_vec()),
             (c1s2.stream_id(), data[1..].to_vec()),
         ];
-        block_on(|res| atomic.write_atomic(updates, res))?;
+        block_on(|res| multi.write_atomic(updates, res))?;
         assert_eq!(c1s1_read.snapshot()?.read_to_end(), data[..1].to_vec());
         assert_eq!(c1s2_read.snapshot()?.read_to_end(), data[1..].to_vec());
 
         // Cannot write to streams not specified during construction.
         let (c1s3, _) = client1.create_or_load("3")?;
```

Comment:
might be worth adding a seal test here as well?

Reply:
Done